tests/test_fmi.py

```diff
@@ -1155,6 +1155,71 @@ def test_simulation_without_initialization(self):
         opts["initialize"] = False
 
         nose.tools.assert_raises(FMUException, model.simulate, options=opts)
+    
+    @testattr(stddist = True)
+    def test_simulation_with_syncronization_exception_ME(self):
+        model = Dummy_FMUModelME2([], "bouncingBall.fmu", os.path.join(file_path, "files", "FMUs", "XML", "ME2.0"), _connect_dll=False)
+        opts = model.simulate_options()
+        opts["synchronize_simulation"] = "Hej"
+        
+        nose.tools.assert_raises(FMUException, model.simulate, options=opts)
```

Comment:
I think it is better if we assert that it raises fmi.InvalidOptionException and not the one the exception extends. I also propose adding a docstring to the tests for maintainability.

Reply:
Agreed, fixed.